### ISSUE TITLE
Less_Cache filemtime(): stat fails on deleted files

### ISSUE THREAD

**Issue by Svish** [oyejorge/less.php#351]
When I delete a file, even though I have removed it from any `@import` statements, I get this crash:

> filemtime(): stat failed for .../src/_less/gone.less

Should maybe be added a `if(file_exists( ... ` or something somewhere? 🤔

**Comment by kylekatarnls** [comment]
Hi, I fixed this bug in my fork, could you try to require `kylekatarnls/less.php` instead of `oyejorge/less.php` and tell me if it works for you?

Thanks.

**Comment by Svish** [comment]
@kylekatarnls Wow, that's lot of changes in that commit... have you changed all the whitespace or something?

**Comment by kylekatarnls** [comment]
Yep, it's the Ctrl + Alt + L of PhpStorm (very close to what StyleCI check). I plan progressively to apply the coding style standards in the fork to unify the code.

**Comment by Svish** [comment]
Anyways, just replaced the `Cache.php` file with the one from your fork, removed an `@import` *and* the file, and it did not crash. 👍 So this one seems to be a win .😀

**Comment by Svish** [comment]
But will this fix get merged back into this one though?

**Comment by kylekatarnls** [comment]
The last code change in this project is quite old. I propose many changes but still no answer from the owner. I'm not sure he's still interested in maintaining this. But there is absolutely no problem with using my fork instead and replace it with the original project if it's updated later. That's the git spirit and the licence allows it.
